rpc_jobs/irr_role_test.yml

```diff
@@ -0,0 +1,102 @@
+- project:
+    name: 'RPC-IRR-Jobs'
+    # Note: branch is the branch for periodics to build
+    #       branches is the branch pattern to match for PR Jobs.
+    series:
+      - master:
+          branch: master
+          branches: "master"
+    image:
+      - xenial:
+          IMAGE: "Ubuntu 16.04 LTS (Xenial Xerus) (PVHVM)"
+      - trusty:
+          IMAGE: "Ubuntu 14.04 LTS (Trusty Tahr) (PVHVM)"
+    context:
+      - base
+    ztrigger:
+      - pr:
+          CRON: ""
+      - periodic:
+          branches: "do_not_build_on_pr"
+          NUM_TO_KEEP: 10
+    jobs:
+      - 'RPC-IRR_{series}-{image}-{context}-{ztrigger}'
+
+- job-template:
+    # DEFAULTS
+    STAGES: >
+      Allocate Resources,
+      Connect Slave,
+      Prepare Deployment,
+      Run IRR Tests,
+      Cleanup,
+      Destroy Slave
+    branch: master
+    NUM_TO_KEEP: 30
+
+    # TEMPLATE
+    name: 'RPC-IRR_{series}-{image}-{context}-{ztrigger}'
+    project-type: workflow
+    concurrent: true
+    properties:
+      - build-discarder:
+          num-to-keep: "{NUM_TO_KEEP}"
+      # IRR REPOS ARE ADDED HERE
```

Comment:
Ah, if that is the case then there will need to be a repo axis. I don't think that the property can be set in the axis though, can it? If not then how would one implement the axis so that it filters properly?